web-client/src/views/StartCaseUpdated/UpdatedFilePetitionStep1.tsx

```diff
@@ -0,0 +1,721 @@
+import {
+  BUSINESS_TYPES,
+  ESTATE_TYPES,
+  MAX_FILE_SIZE_MB,
+  OTHER_TYPES,
+} from '@shared/business/entities/EntityConstants';
+import { Button } from '@web-client/ustc-ui/Button/Button';
+import { ContactPrimaryUpdated } from '@web-client/views/StartCase/ContactPrimaryUpdated';
+import { ContactSecondaryUpdated } from '@web-client/views/StartCase/ContactSecondaryUpdated';
+import { FormGroup } from '@web-client/ustc-ui/FormGroup/FormGroup';
+import { InfoNotificationComponent } from '@web-client/views/InfoNotification';
+import { StateDrivenFileInput } from '@web-client/views/FileDocument/StateDrivenFileInput';
+import { UpdatedFilePetitionButtons } from '@web-client/views/StartCaseUpdated/UpdatedFilePetitionButtons';
+import { WarningNotificationComponent } from '@web-client/views/WarningNotification';
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { sequences, state } from '@web-client/presenter/app.cerebral';
+import { useValidationFocus } from '@web-client/views/UseValidationFocus';
+import React from 'react';
+import classNames from 'classnames';
+
+/* eslint-disable max-lines */
```

Comment:
Any reason not to split out components into separate files in a FilePetitionStep1 folder?